configuration/shared_subscriptions.md

```diff
@@ -22,3 +22,23 @@ shared_subscription_policy = prefer_local
 
 When a messages is being delivered to subscribers of a shared subscription, the message will be delivered to an online subscriber if possible, otherwise the message will be delivered to an offline subscriber.
 
+## Examples
+
+**Subscriptions**
+*Note: When subscribing to a shared topic, make sure to escape the `$` 
+```bash
+mosquitto_sub -h mqtt.example.io -p 1883 -q 2 -t $share/group/topicname
+mosquitto_sub -h mqtt.example.io -p 1883 -q 2 -t $share/group/topicname/#
+```
+So, for dash on ubuntu bionic
+```bash
+mosquitto_sub -h mqtt.example.io -p 1883 -q 2 -t \$share/group/topicname
+mosquitto_sub -h mqtt.example.io -p 1883 -q 2 -t \$share/group/topicname/#
+```
+
+**Publishing**
+*Note: When publishing to a shared topic, do not include the phrase **$share/group/** as part of the publish topic name*
```

Comment:
Better wording: instead of "phrase" I'd call it "prefix"

Reply:
Can you emphasize the topic using backticks instead of asterisks like `$share/group/`  instead of **$share/group/**

Reply:
Yes to both.